gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/config/AmqpConfiguration.java

```diff
@@ -0,0 +1,78 @@
+package uk.nhs.adaptors.pss.translator.config;
+
+import static org.springframework.util.StringUtils.isEmpty;
+
+import org.apache.qpid.jms.JmsConnectionFactory;
+import org.apache.qpid.jms.JmsDestination;
+import org.apache.qpid.jms.message.JmsMessageSupport;
+import org.apache.qpid.jms.policy.JmsRedeliveryPolicy;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jms.config.DefaultJmsListenerContainerFactory;
+import org.springframework.jms.config.JmsListenerContainerFactory;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.jms.support.converter.MappingJackson2MessageConverter;
+import org.springframework.jms.support.converter.MessageConverter;
+
+@Configuration
+public class AmqpConfiguration {
+    @Bean
+    public MessageConverter jsonMessageConverter() {
+        return new MappingJackson2MessageConverter();
+    }
+
+    @Bean("pssQueueConnectionFactory")
+    public JmsConnectionFactory jmsConnectionFactoryPssQueue(PssQueueProperties properties) {
+        JmsConnectionFactory factory = new JmsConnectionFactory();
+
+        factory.setRemoteURI(properties.getBroker());
+
+        if (!isEmpty(properties.getUsername())) {
```

Comment:
IntelliJ shows that this `isEmpty` method is deprecated. Maybe you could use `properties.getUsername().isEmpty()` instead? Here and in others if statements in this class.